tools/run_tests/sanity/banned_functions.py

```diff
@@ -209,14 +209,6 @@
         "./src/core/util/subprocess_windows.cc",
         "./src/core/util/time_precise.cc",
         "./src/core/util/windows/log.cc",
-        "./src/core/xds/grpc/xds_client_grpc.cc",
-        "./src/core/xds/grpc/xds_cluster.cc",
-        "./src/core/xds/grpc/xds_endpoint.cc",
-        "./src/core/xds/grpc/xds_listener.cc",
-        "./src/core/xds/grpc/xds_route_config.cc",
-        "./src/core/xds/xds_client/xds_api.cc",
-        "./src/core/xds/xds_client/xds_client.cc",
-        "./src/core/xds/xds_client/xds_client_stats.cc",
         "./src/php/ext/grpc/call_credentials.c",
         "./src/php/ext/grpc/channel.c",
```

Comment:
This is a list of files where use of gpr_log is allow - listed.
As we clean up the files, we keep deleting entries from here to prevent backsliding.